commands/deploy.go

```diff
@@ -131,7 +140,9 @@ func (c *DeployCmd) Run() (err error) {
 	artifactSizeMB := stat.Size() / 1024 / 1024
 	log.Printf("[DEBUG] Upload artifact is %dMB (%d bytes) large", artifactSizeMB, stat.Size())
 	s = NewSpinner(fmt.Sprintf("Uploading app (%dMB)...", artifactSizeMB))
-	s.Start()
+	if !(c.Quiet){
```

Comment:
Instead of checking Quiet when we start the spinner, I think a better pattern would be for us to modify our `NewSpinner` function to return a no-op spinner that matches the `spinner.Spinner` interface, but does nothing when start/stop are called. Then we don't have to remember to make this check if we add more spinners in the future.